extensions/blocks/conversation/attributes.js

```diff
@@ -0,0 +1,9 @@
+export default {
+	participants: {
+		type: "array",
+	},
+	showTimeStamp: {
+		type: "boolean",
```

Comment:
Small nitpick (I'm actually surprised you were able to commit this, I would expect Prettier to change this for you automatically).

```suggestion
		type: 'array',
	},
	showTimeStamp: {
		type: 'boolean',
```